chef/cookbooks/ceilometer/recipes/common.rb

```diff
@@ -66,7 +66,7 @@
   bind_host = admin_address
   bind_port = node[:ceilometer][:ha][:ports][:api]
 else
-  bind_host = "0.0.0.0"
+  bind_host = node[:ceilometer][:api][:host]
```

Comment:
I did not like the hard-coded value here. I am open to revert it, this is not important to me.